2-positioning/styles.css

```
@@ -0,0 +1,48 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+header {
```

```suggestion
.header {
```
Always prefer classes over tag names